### PR TITLE
[4.0] ES6 please, sampledata.js

### PR DESCRIPTION
**Pull request by dgrammatiko**
Pull Request for Issue # .

### Summary of Changes
Refactor to ES6

### Testing Instructions
Check if sample data is installed correctly

### CODE REVIEW

File: build/media_src/mod_sampledata/js/sampledata-process.es6.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      return;
+    }
+
+    if (element.getAttribute('data-processed')) {
+      alert(Joomla.JText._('MOD_SAMPLEDATA_ITEM_ALREADY_PROCESSED'));
+      return;
+    }
+
+    // Make sure that use run this not by random clicking on the page links
+    if (!confirm(Joomla.JText._('MOD_SAMPLEDATA_CONFIRM_START'))) {
```
[the remark this answers is not in the document]
Comment (dgrammatiko) [reply build/media_src/mod_sampledata/js/sampledata-process.es6.js]: Nope, deal with it

File: build/media_src/mod_sampledata/js/sampledata-process.es6.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        if (response.success && response.data && response.data.length > 0) {
+          const progress = document.querySelector(`.sampledata-progress-${type} .progress-bar`);
+
+          // Display all messages that we got
+          response.data.forEach((value) => {
+            if (value === null) {
+              return;
+            }
+
+            success = value.success;
```
[the remark this answers is not in the document]
Comment (dgrammatiko) [reply build/media_src/mod_sampledata/js/sampledata-process.es6.js]: Nope, deal with it

File: build/media_src/mod_sampledata/js/sampledata-process.es6.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (wilsonge) [reply build/media_src/mod_sampledata/js/sampledata-process.es6.js]: then add an es-lint-ignore please to both!

[the remark this answers is not in the document]
Comment (dgrammatiko) [reply build/media_src/mod_sampledata/js/sampledata-process.es6.js]: @wilsonge done

**Pull request merged**